ocs_ci/ocs/ocp.py

```diff
@@ -1115,21 +1115,31 @@ def confirm_cluster_operator_version(target_version, cluster_operator):
     return False
 
 
-def upgrade_ocp(image_path, image):
+def upgrade_ocp(upgrade_vars):
```

Comment:
I like more the previous approach with specific arguments instead of dictionary with data as it's more clear to me to see specifically passed the arguments and have None value as default if the parameter is not always needed.

And if you would like to use the function you can still have data defined in the dictionary from place you call this function and just pass it like:
upgrade_ocp(**upgrade_vars)


Or what is behind this to rather take only one dictionary?

Reply:
as default, both parameters is not needed, so im using dict_name.get()

Reply:
> as default, both parameters is not needed, so im using dict_name.get()

Hence you can use None as default values:
image_path=None, image=None and others the same if they are not mandatory and then you can do :  
```python
if image_path:
    do something

Reply:
fixed